test/app-luatest/gh_6200_lua_cjson_escapes_slash_test.lua

```diff
@@ -53,25 +53,21 @@ local function server_test_json_new_encode()
     -- Test that '/' is escaped with 'old' setting.
     t.assert_equals(json.encode('/'), [["\/"]])
 
-    -- Create new serializer and check that it has correct defaults
-    -- and doesn't change behavior.
+    -- Test that a serializer created with json.new() follows
```

Comment:
This feature (configuring forward slash escape behavior per json serializer) looks unintended to me. I reckon it was introduced solely for the compat's sake, because the `encode_escape_forward_slash` JSON serializer option wasn't documented or even mentioned anywhere, not even in the commit message (see commit 1436601cbf8a6796bc0c9493bafd7206fbf50ed8).

Since we assume that escaping the forward slash character is a bug and are going to drop this functionality completely after a couple major releases, I don't think we should preserve this serializer option.